data/json/terrain.json

```diff
@@ -29,14 +28,214 @@
         "trap": "tr_ledge",
         "flags": ["TRANSPARENT", "NO_FLOOR"],
         "examine_action": "climb_down"
+    },{
+		"type": "terrain",
+		"id": "t_wframe",
+		"name": "wooden building frame",
+		"description": "A standard 16\" stud wooden building frame, the mainstay of modern home construction.  While it will support a roof, unless construction is completed it will offer little protection against intruders.",
+		"symbol": "#",
+		"color": "brown",
+		"move_cost": 4,
+		"flags": ["SUPPORTS_ROOF", "THIN_OBSTACLE", "WALL", "TRANSPARENT", "PERMEABLE", "FLAMMABLE"],
+		"bash": {
+			"str_min": 10, "str_max": 20,
+            "sound": "crash!",
+            "sound_fail": "bash!",
+            "ter_set": "t_null"
+			}
+	},{
+		"type": "terrain",
+		"id": "t_sframe",
+		"name": "steel building frame",
+		"description": "A steel building frame, the mainstay of modern large scale construction.  Increadably stron structuraly but provides no protection from the oustide.",
```

Comment:
I suggest you rename this to steel column and mention its part of a frame in the description. The steel frames of a building  are too long to fit in a single tile, they are normally no smaller than 6m long.

It might also be rather complex to make our current  buildings use them sensibly, they shouldn't be one right next to each other like normal walls.